x11-plugins/wmitime/wmitime-0.5-r1.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit toolchain-funcs
+
+DESCRIPTION="Overglorified clock dockapp w/time, date, and internet time"
+HOMEPAGE="https://www.dockapps.net/wmitime"
+SRC_URI="https://dev.gentoo.org/~voyageur/distfiles/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~sparc ~x86"
+
+RDEPEND=">=x11-libs/libdockapp-0.7:=
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXpm"
+DEPEND="${RDEPEND}"
+
+src_compile() {
+	emake CC="$(tc-getCC)" \
+		CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}"
+}
+
+src_install() {
+	emake DESTDIR="${D}" PREFIX=/usr install
+	dodoc BUGS CHANGES README
```

Comment:
Cannot we move this to DOCS and remove the whole src_install() section?

Reply:
Well i could move the docs into the DOCS variable, but i still need the emake as the default installation folder would be ```/usr/local/``` instead of ```/usr```

Reply:
Ok so probably moving to DOCS and calling einstalldocs is a better approach with these hardcoded Makefiles.

Reply:
i've replaced the ```dodoc``` line with ```einstalldocs```.
As ```einstalldocs``` would install the default set of docs anyway i didn't had to set the ```DOCS``` var after all :)